Update quick start guide (#2198)
Melledy/Grasscutter: README.md

**Note**: For support please join our [Discord](https://discord.gg/T5vZU6UyeG).

> • Get Java 17: https://www.oracle.com/java/technologies/javase/jdk17-archive-downloads.html

Reviewer: ```suggestion
### Quick Start (automatic)

• Get Java 17: https://www.oracle.com/java/technologies/javase/jdk17-archive-downloads.html
```

> • Get MongoDB Community Server: https://www.mongodb.com/try/download/community

Reviewer: ```suggestion
• Get [MongoDB Community Server](https://www.mongodb.com/try/download/community)
```

> • Get Genshin version 3.7 (Genshin 3.7 client can be found here if you don't have it): https://github.com/MAnggiarMustofa/GI-Download-Library/blob/main/GenshinImpact/Client/3.7.0.md)

Reviewer: ```suggestion
• Get game version REL3.7 (3.7 client can be found here if you don't have it): https://github.com/MAnggiarMustofa/GI-Download-Library/blob/main/GenshinImpact/Client/3.7.0.md)
```

> • Download the latest Cultivation version. Install the .msi: https://github.com/Grasscutters/Cultivation/releases/latest

Reviewer: ```suggestion
• Download the [latest Cultivation version](https://github.com/Grasscutters/Cultivation/releases/latest). Use the `.msi` installer.
```

• After opening Culivation (as admin), press the download button in the upper right corner. 

> • Click Download All-in-One

Reviewer: ```suggestion
• Click `Download All-in-One`
```

• Click the gear in the upper right corner

> • Set the game Install path to where Genshin Impact\Genshin Impact game\GenshinImpact.exe is.

Reviewer: ```suggestion
• Set the game Install path to where your game is located.
```

> • Set the Custom Java Path to C:\Program Files\Java\jdk-17\bin\java.exe

Reviewer: ```suggestion
• Set the Custom Java Path to `C:\Program Files\Java\jdk-17\bin\java.exe`
```
